Tests/APITesters/AllAPITests/SwiftAPITester/VirtualCurrencyAPI.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  VirtualCurrencyAPI.swift
+//  SwiftAPITester
+//
+//  Created by Will Taylor on 2/28/25.
+//
+
+import Foundation
+import RevenueCat
+
+func checkVirtualCurrency(virtualCurrency: VirtualCurrency) {
+    let balance: Int = virtualCurrency.balance
+    let name: String = virtualCurrency.name
+    let code: String = virtualCurrency.code
+    let serverDescription: String? = virtualCurrency.serverDescription
+
+    // Ensure that we don't break the nullability of serverDescription.
+    // Just ensuring that it's a String? isn't enough since a String can be automatically casted
+    // to a String?, like so:
+    //
+    // let string1: String = ""
+    // let string2: String? = string1
+    var serverDescriptionNullabilityTest: String? = virtualCurrency.serverDescription
+    serverDescriptionNullabilityTest = nil
```

Comment:
Hmm I'm unsure how this test actually confirms that `virtualCurrency.serverDescription` still remains nullable... Can you give more details?

IMO, the best way to check this would be by creating an instance of a `VirtualCurrency` here... Wdyt?

Reply:
Yeah, I suppose that `virtualCurrency.serverDescription` could be a `String`, and that would still successfully cast to a `String`. 

Unfortunately, there's no public initializer for `VirtualCurrency` besides `init(from:)`, so I think I'll just remove this for now and keep the `let serverDescription: String? = virtualCurrency.serverDescription`